fmc/README.md

```diff
@@ -113,8 +113,8 @@ fields may not be changed or removed). Table revisions with different Major Vers
 | fht_minor_ver         | 2            | ROM, FMC   | Minor version of FHT. Initially written by ROM but may be changed to a higher version by FMC.            |
 | manifest_load_addr    | 4            | ROM        | Physical base address of Manifest in DCCM SRAM.                                                          |
```

Comment:
@vsonims, would it make sense for this to also be in the DV, or should this stay as physical address?

Reply:
Let's merge with what ROM already stores in the DV and sort this out later as an issue assigned to Vishal.